cpp/CMakeLists.txt

```diff
@@ -18,6 +18,8 @@
 cmake_minimum_required(VERSION 2.7)
 project(arrow)
 
+set(ARROW_VERSION "0.2.0")
```

Comment:
As an alternative, we could add a `.arrowversion` file to the root of the repository that is updated with the release scripts. We do this in the release scripts of `parquet-cpp`. The main problem in Arrow is that we use the Maven release plugin and we haven't yet found a hook to write out the version string in the correct commit in a non-xml file.